css-view-transitions-2/Overview.bs

```diff
@@ -616,186 +563,391 @@ Prepend this to the [=Perform pending transition operations=] algorithm given a
+
+## Examples ## {#vt-class-example}
+
+<div class="example">
+This example creates a transition with each box participating under its own name, while applying
+a 1-second duration to the animation of all the boxes:
+
+```html
+<div class="box" id="red-box"></div>
+<div class="box" id="green-box"></div>
+<div class="box" id="yellow-box"></div>
+```
+
+```css
+div.box {
+	view-transition-class: any-box;
+	width: 100px;
+	height: 100px;
+}
+#red-box {
+	view-transition-name: red-box;
+	background: red;
+}
+#green-box {
+	view-transition-name: green-box;
+	background: green;
+}
+#yellow-box {
+	view-transition-name: yellow-box;
+	background: yellow;
+}
+
+/* The following style would apply to all the boxes, thanks to 'view-transition-class' */
+::view-transition-group(*.any-box) {
+	animation-duration: 1s;
+}
+```
+</div>
+
+## The 'view-transition-class' property ## {#view-transition-class-prop}
+
+	<pre class=propdef>
+	Name: view-transition-class
+	Value: none | <<custom-ident>>+
+	Initial: none
+	Inherited: no
+	Percentages: n/a
+	Computed Value: as specified
+	Animation type: discrete
+	</pre>
+
+	The 'view-transition-class' can be used to apply the same style rule to multiple [=named view transition pseudo-elements=] which may have a different 'view-transition-name'.
+	While 'view-transition-name' is used to match between the element in the old state with its corresponding element in the new state, 'view-transition-class' is used
+	only to apply styles using the view transitionpseudo-elements
+	(''::view-transition-group()'', ''::view-transition-image-pair()'', ''::view-transition-old()'', ''::view-transition-new()'').
+
+	Note that 'view-transition-class' by itself doesn't mark an element for capturing, it is only used as an additional
+	way to style an element that already has a 'view-transition-name'.
+
+	<dl dfn-type=value dfn-for=view-transition-class>
+		: <dfn>none</dfn>
+		:: No class would apply to the [=named view transition pseudo-elements=] generated for this element.
+
+		: <dfn><<custom-ident>>+</dfn>
+		:: All of the specified <<custom-ident>> values (apart from <css>none</css>) are applied when used in [=named view transition pseudo-element=] selectors.
+			<css>none</css> is an invalid <<custom-ident>> for 'view-transition-class', even when combined with other <<custom-ident>>s.
+
+			Note: If the same 'view-transition-name' is specified for an element both in the old and new states of the transition,
+			only the 'view-transition-class' values from the new state apply. This also applies for cross-document view transitions:
+			classes from the old document would only apply if their corresponding 'view-transition-name' was not specified in the new document.
+	</dl>
+
+## Additions to named view transition pseudo-elements ## {#pseudo-element-class-additions}
+
+	The [=named view transition pseudo-elements=]
+	(''view-transition-group()'', ''view-transition-image-pair()'', ''view-transition-old()'', ''view-transition-new()'')
```

Comment:
```suggestion
```

No need to re-list all the named elements, that's what the definition link is for. :)